rosetta/validator/requests.go

```diff
@@ -41,33 +41,47 @@ const (
 	symbolField      = "symbol"
 	transactionField = "transaction"
 	signaturesField  = "signatures"
+
+	blockchainFailTag = "blockchain"
+	networkFailTag    = "network"
+)
+
+// Error descriptions for common errors.
+const (
+	unknownBlockchain = "network identifier has unknown blockchain field"
+	unknownNetwork    = "network identifier has unknown network field"
 )
 
-func newRequestValidator() *validator.Validate {
+func (v *Validator) newRequestValidator() *validator.Validate {
 
-	v := validator.New()
+	vl := validator.New()
```

Comment:
Not super fond of the `v1` naming tbh :p WDYT of calling it `valid` or smth like that instead? Or renaming the pointer receiver instead?